github/repos_pages.go

```diff
@@ -93,6 +93,25 @@ func (s *RepositoriesService) GetLatestPagesBuild(owner string, repo string) (*P
 	return build, resp, err
 }
 
+// GetPageBuild fetches the specific build information for a GitHub pages site.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/pages/#list-a-specific-pages-build
+func (s *RepositoriesService) GetPageBuild(owner string, repo string, ID int) (*PagesBuild, *Response, error) {
```

Comment:
Can you please lower-case "id" here?

Also, I realize the examples above don't use the Go idiomatic style, which is my fault for not catching before, but can we change `(owner string, repo string, ID int)` to `(owner, repo string, id int)` please?  This more closely matches Go idiomatic style and also the rest of this repo.

Feel free to change the above function signatures too for more consistency if you wish.